### PR TITLE
Vaults to Spaces step 6

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

This PR implements various polish for the Vaults to Spaces transition:
- Default restriction based on the CTAs to create a vault
- Use connector name in "Add data..." CTA
- Refresh content nodes when updating data

### CODE REVIEW

**Review by JulesBelveze — APPROVED** [review]
LGTM

File: front/components/vaults/CreateOrEditVaultModal.tsx [hunk 1]
```diff
@@ -76,9 +78,7 @@ export function CreateOrEditVaultModal({
   const [isSaving, setIsSaving] = useState(false);
   const [showDeleteConfirmDialog, setShowDeleteConfirmDialog] = useState(false);
   const [isDeleting, setIsDeleting] = useState(false);
-  const [isRestricted, setIsRestricted] = useState(
-    vault?.isRestricted ?? false
-  );
+  const [isRestricted, setIsRestricted] = useState(false);
```
[the remark this answers is not in the document]
Comment (flvndvd) [reply front/components/vaults/CreateOrEditVaultModal.tsx:81]: Cause those default values are not relevant, they are being set by a `useEffect` on line 94.

[the remark this answers is not in the document]
Comment (JulesBelveze) [reply front/components/vaults/CreateOrEditVaultModal.tsx:81]: Yeah agree but your code is equivalent to saying "restricted is false if defaultRestricted isn't passed", right? 😛 
No biggie, same thing in the end

[the remark this answers is not in the document]
Comment (flvndvd) [reply front/components/vaults/CreateOrEditVaultModal.tsx:81]: I meant this default value is always superseeded, right?

**Pull request merged**